apps/web/src/styles/theme.ts

```diff
@@ -45,6 +47,12 @@ const theme = extendTheme({
   components: {
     Text: {
       sizes,
+      variants: {
+        bold: {
+          fontWeight: "600",
+          color: "var(--chakra-colors-gray-900)",
```

Comment:
that's not how we should define the colours. please check [this](https://v2.chakra-ui.com/docs/styled-system/semantic-tokens#conditional-tokens)